modulemd/tests/test-modulemd-moduleindex.c

```diff
@@ -409,15 +409,9 @@ module_index_test_stream_upgrade (void)
   gboolean ret;
   g_autoptr (ModulemdModuleIndex) index = NULL;
   g_autoptr (ModulemdModuleStream) stream = NULL;
-  g_autoptr (ModulemdDependencies) dep = NULL;
   g_autoptr (GError) error = NULL;
-  ModulemdModuleStreamVersionEnum default_mdv;
-
-  /* get and save current default stream mdversion */
-  default_mdv = modulemd_get_default_stream_mdversion ();
 
   /* Construct a v1 Index with some objects */
```

Comment:
```suggestion
  /* Construct an Index with some objects */
```

Since Index's are no longer locked to a specific mdversion, it's no v1-specific